frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -409,14 +411,26 @@ export class HtmlDocument extends React.Component {
   handleSaveHtmlDocument = async () => {
     const { state, props } = this
 
+    let newDocumentForApiWithMention
+    try {
+      newDocumentForApiWithMention = wrapMentionInSpanTag(state.content.raw_content)
+    } catch (e) {
+      if (e === EXCEPTION_MENTION_PARSING) {
```

Comment:
This means you raise a constant?
Wouldn't an Error object (or a class inheriting from Error) be cleaner? (and then you can test with `instanceof`).
No hard opinion on this, you can skip this comment.

Reply:
I agree with @raphj here: the `Error` object is more appropriate as it also includes a message + relevant information to the raised value.

Reply:
Changed